packages/manager/apps/web/client/app/domain-operation/operation-table/operation-table.html

```diff
@@ -0,0 +1,225 @@
+    data-on-sort-change="$ctrl.onSortChange($sort)"
+    data-columns="$ctrl.columnsConfig"
+    data-on-columns-parameters-change="$ctrl.onColumnChange(id, columns)"
+    data-page="{{ $ctrl.paginationNumber }}"
+    customizable
+>
+    <oui-datagrid-column
+        data-title="::'domain_operations_table_header_domain' | translate"
+        data-property="zone"
+        data-type="string"
+        data-sortable
+        data-filterable
+        data-searchable
+    >
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title="::'domain_operations_table_header_function' | translate"
+        data-property="function"
+        data-type="options"
+        data-type-options="$ctrl.operationFunctionColumnOptions"
+        data-filterable
+        data-sortable
+    >
+        <span
+            data-ng-bind="'domain_operations_nicOperation_' +  $row.function | translate"
+        ></span>
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title="::'domain_operations_table_header_comment' | translate"
+        data-property="comment"
+        data-type="string"
+        data-type-options="$ctrl.domainSuspensionStateColumnOptions"
+        data-filterable
+        data-sortable
+    >
+    </oui-datagrid-column>
+
+    <oui-datagrid-column
+        data-title="::'domain_operations_table_header_todoDate' | translate"
+        data-property="todoDate"
+    >
+        {{ $row.todoDate | date:'short' }}
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title="::'domain_operations_table_header_lastUpdate' | translate"
+        data-property="lastUpdate"
+    >
+        {{ $row.lastUpdate | date:'short' }}
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title="::'domain_operations_table_header_doneDate' | translate"
+        data-property="doneDate"
+    >
+        {{ $row.doneDate | date:'short' }}
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title="::'domain_operations_table_header_status' | translate"
+        data-property="status"
+    >
+        <span
+            class="oui-badge"
+            data-ng-class="{
+                'oui-badge_info': $row.status === 'todo' || $row.status === 'doing' || $row.status === 'cancelled',
+                'oui-badge_error': $row.status === 'error',
+                'oui-badge_success': $row.status === 'done'
+              }"
+            data-ng-bind="'domain_operations_statusOperation_' +  $row.status | translate"
+        >
+        </span>
+        <span
+            data-oui-tooltip="{{:: 'domain_operations_no_action' | translate }}"
+            data-oui-tooltip-enable="{{!$row.canCancel && !$row.canRelaunch && !$row.canAccelerate}}"
+            data-oui-tooltip-placement="left"
+        >
+            <oui-action-menu
+                data-compact
+                data-disabled="!$row.canCancel && !$row.canRelaunch && !$row.canAccelerate && $row.function !== 'DomainIncomingTransfer'"
```

Comment:
`DomainIncomingTransfer` appears a lot of time, so you might create a constant

Reply:
Same here, legacy code I only moved